src/main/java/com/mineboundteam/minebound/item/MyrialSword.java

```diff
@@ -0,0 +1,36 @@
+package com.mineboundteam.minebound.item;
+
+import com.mineboundteam.minebound.entity.MyrialSwordEntity;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.SwordItem;
+import net.minecraft.world.item.Tier;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.NotNull;
+
+public class MyrialSword extends SwordItem {
+    MyrialSwordEntity myrialSwordEntity;
+    public int state = 0;
+
+    public MyrialSword(Tier tier, int attackDamageModifier, float attackSpeedModifier, Properties properties) {
+        super(tier, attackDamageModifier, attackSpeedModifier, properties);
+    }
+
+    @Override
+    public @NotNull InteractionResultHolder<ItemStack> use(@NotNull Level level, @NotNull Player player, @NotNull InteractionHand interactionHand) {
```

Comment:
Use is only at the start of using the item, not its duration. This should be onUsingTick if it's going to be done this way